portfolio/src/main/java/com/google/sps/data/CommentLoader.java

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.lang.Integer;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Arrays;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.sps.data.Comment;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.gson.Gson;
+
+/**
+ * A utility class for loading comments from datastore.
+ */
+public final class CommentLoader {
+  private DatastoreService datastore;
+  private List comments;
```

Comment:
Why do you need a class member for `comments`? It could be converted to a temporary variable.

Reply:
I though it was better memory-wise (initializing the array only once, and not every time we call the function), but I don't know if it has a meaningful effect.. Done.